geode-docs/basic_config/data_entries_custom_classes/managing_data_entries.html.md.erb

```diff
@@ -60,36 +60,90 @@ You can also use the `gfsh put` command to add entries to a region, and the `get
 
 If you want only to create the entry (with a null value and with method failure if the entry already exists), use `Region.create` instead.
 
-## <a id="managing_data_entries__section_7578349EA26A4621B732FE851D71A84F" class="no-quick-link"></a>Batch Operations (getAll, putAll, removeAll)
+## <a id="getAll_method" class="no-quick-link"></a>The getAll Operation
 
-<%=vars.product_name%> provides three APIs to perform batch operations on multiple region entries:
+The batch operation `Region.getAll`
+takes a collection of keys and returns a `Map` of key-value pairs for
+the provided keys. If a given key does not exist in the region, then that key's value in the returned map will be null.
 
--   `Region.getAll`
--   `Region.putAll`
--   `Region.removeAll`
+## <a id="putAll_method" class="no-quick-link"></a>The putAll Operation
 
-The `getAll` method takes a collection of keys and returns a `Map` of values for the provided keys. If a given key does not exist in the region, then that key's value in the returned map will be null.
+The batch operation `Region.putAll`
+takes a `Map` of key-value pairs, puts them into the cache,
+and distributes them in a single operation.
 
-The `putAll` method takes a `Map` of key-value pairs and puts them into the cache and distributes them in a single operation.
 
-**Example:**
+The updates to the cache are done individually in the order in which
+they were placed in the `Map`.
```

Comment:
Actually the order of processing for keys in the map is not guaranteed. But we don't have to mention it.

Reply:
Nice to know.  That info was already in our docs.  Must've been there and incorrect for a long time!  Is the lack of an ordering guarantee also in effect for the other bulk operations?

Reply:
Same for removeAll. It depends on what kind of map user is using. For example, if user is using LinkedHashMap, the order of keys will be maintained. But we did not enforce the user to have to use LinkedHashMap. Our API accepts any Map implementation.